modules/es-extensions/publisher/extensions/app/greg-diff-api/modules/governance-diff-api.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Thanks @thushara35 , I have updated the copyright year in [this commit](https://github.com/wso2/product-greg/pull/854/commits/77baf80d16294c4df8bed39d18eb41cd70fa0fd0)